src/components/Metrics/Metrics-i18n.json

```diff
@@ -1,5 +1,6 @@
 {
   "metrics": {
+    "axis-label-bytes": "Bytes",
     "axis-label-time": "Time",
     "used_disk_space": "Used disk space",
     "used_disk_space_popover_header": "Used disk space chart",
```

Comment:
I don't actually see this text in the popover for this chart 🤔  but is it possible to remove the text "chart" from all of the `_popover_header` strings? We should just repeat the name of the chart as the popover header.

Reply:
Sure!

Reply:
Done

Reply:
Btw that string is used to label the Y axis of the bytes related charts in the topic metrics card.

I added a tool to Storybook to disable the translations so to see the keys directly. It works like this
![Kapture 2021-11-30 at 17 57 23](https://user-images.githubusercontent.com/966316/144092470-2e85183a-d82c-4e10-943e-e7aa7dada9f3.gif)